src/translations/en.json

```diff
@@ -112,6 +112,7 @@
   "claim.edit.services.fetchingPricelist": " (fetching prices list…)",
   "claim.edit.services.totalClaimed": "Total Claimed: {totalClaimed}",
   "claim.edit.services.totalApproved": "Total Approved: {totalApproved}",
+  "claim.edit.services.MaxApproved": "Quantity set is more than quantity approved for this element. You will be reimbursed only for {totalApproved} element(s)",
```

Comment:
@dragos-dobre I am not sure, but haven't we blocked this option already? You can't have more quantity than approved.